src/toast/ops/elevation_noise.py

```diff
@@ -115,55 +155,69 @@ def _exec(self, data, detectors=None, **kwargs):
             log.error(msg)
             raise RuntimeError(msg)
 
-        for ob in data.obs:
-            focalplane = ob.telescope.focalplane
+        for obs in data.obs:
```

Comment:
I have some other changes to the `ElevationNoise` operator (plus adding unit tests) in a branch where I am working through some HDF5 unit test failures.  So I will have to manually merge these changes.  Is there a reason you change the iteration variable `ob` --> `obs`?  Just preference?  I guess it doesn't matter since I'll have to merge this line-by-line with the other changes, but will try to make sure that the name is used consistently when doing that.

Reply:
I cannot recall what the logic behind changing the name was... However, we should pick either `ob` or `obs` as the canonical name and use it throughout. TOAST-2 is pretty consistent about using `obs` but sizable fraction of TOAST-3 switched to `ob`.

Reply:
Correct, much of the code was switched to `ob` to avoid confusion with the `Data.obs` member.